src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,43 @@
+@import "../utils/reset";
+@import "../utils/variables";
+@import "../utils/extends";
+
+.nav {
+  text-align: center;
+
+  &__list {
+    @extend .nav;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    list-style: none;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    @extend %link-style;
+    display: flex;
+    padding: 23px 0;
+    color: $main-text-color;
+  }
+
+  &__link:hover,
+  .is-active {
+    position: relative;
+    color: $light-blue-link-color;
+  }
+}
+
+.is-active:after {
+  position: absolute;
+  content: "";
+  width: 100%;
+  height: 4px;
+  left: 0;
+  bottom: 0;
+  background: $light-blue-link-color;
+  border-radius: 8px;
+}
```

Comment:
You should nest '.is-active::after' pseudo-class into '.is-active' (Checklist#10).

Hovered 'nav__link' does not need relative positioning, it should only change the color.